libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -679,9 +693,10 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 		else
 		{
 			YulArity arity = YulArity::fromType(*functionType);
+			m_context.internalFunctionCalledViaPointer(arity);
```

Comment:
Maybe `throughDispatch` instead of `viaPointer`?

Reply:
Done.

Also changed `via_pointer` to `through_dispatch` in test names.